__test__/pet-store.spec.ts

```diff
@@ -212,6 +212,33 @@ describe('Pet Store', () => {
         })
       }).toThrow(RequestValidationError)
     })
+
+    test('It returns defined body content type', () => {
+      expect(chowchow.getDefinedBodyContentType('/pets', 'post'))
+        .toMatchInlineSnapshot(`
+Array [
```

Comment:
looks like there's some formatting issue here.

Reply:
That's actually the formatting created by `jest.toMatchInlineSnapshot`.
I guess it needs this formatting to compare the text.